report/report.tex

```diff
@@ -37,8 +37,30 @@
 \end{IEEEkeywords}
 
 \section{Introduction}
+Digital door locks are becoming more and more common, and as they become more common, the vulnerabilities they suffer from become more and more dangerous. While digital locks are more convenient by offering a keyless entry and exit, they are susceptible to a slew of different attacks, all intended to derive the static PIN that unlocks the digital lock.
+
 \subsection{Problems with traditional keypad entry systems}
 
+Assuming a 4 digit access code, simple wear and tear on the keypad means that the attacker only needs to try 4! (24) combinations at most to bruteforce the PIN, cutting down the original keyspace by almost 98\% (Figure 1).
+
+\begin{figure}[h!]
+  \includegraphics[width=\linewidth]{worn_lock.png}
+  \caption{Worn keypad exposes digits of PIN}
+  \label{fig:wornlock}
+\end{figure}
+
+More active attacks include shoulder surfing or installing inconspicuous pinhole cameras over the lock, revealing to the attacker exactly what the combination is if the victim is unaware of the attack. 
```

Comment:
perhaps: *revealing the exact combination as an unsuspecting victim uses it to unlock the door*